database-plugins/widgets/Database-batchsource.json

```diff
@@ -59,6 +59,9 @@
           "name": "importQuery",
           "widget-attributes": {
             "rows": "4"
+          },
+          "plugin-function": {
+            "method": "getSchema"
```

Comment:
Please update the specification wiki : https://wiki.cask.co/display/CE/UI+JSON+Specification and get it reviewed before making this change.